WIP: Add ProtoSchool links to ipfs.io

terichadbourne opened the pull request:
Working on the multiple-button layout @olizilla suggested but my attempt to mix flexbox and tachyons is failing. I was envisioning 3 buttons across on a wider screen, stacked on mobile, but neither the wrapping or the bottom margin are taking. The CSS I found is all minimized (I'm not experienced with Hugo so may be looking in the wrong place).

olizilla commented:
As far as I am aware, the ipfs.io website does not currently include tachyons, it users yeti.css from &yet, as they were commissed to build it many moons ago. The css is generated from less files, which you can find here https://github.com/ipfs/website/tree/master/less

terichadbourne commented:
Thanks @olizilla. The non-existence of Tachyons seems like a pretty good excuse for it not to be working. 😂

terichadbourne commented:
@fsdiogo & @lidel Helped me get the formatting working but we'd love feedback on the phrasing for these options: 

![image](https://user-images.githubusercontent.com/19171465/61883808-170e8d80-aec9-11e9-93d1-6378513a3f24.png)

jessicaschilling commented:
"look under the hood" is a little bit of an Americanism. What about "Want to dig in a little?" ... that also implies something that's still technical, but not as technical as someone who might want to read the whitepaper.

momack2 commented:
The whitepaper is *super* out of date - might be better to do something  like "want to see where it all began" or "want the technical vision" to indicate that this isn't actually the spec for the project as it exists today, but is a previous conception from 2014.

terichadbourne commented:
@olizilla I believe this three-button approach was your idea. Do you have any wording suggestions before we lock this down?

terichadbourne commented:
Based on feedback I've updated the wording and moved the whitepaper to the right to make it lower priority given its age. 

![image](https://user-images.githubusercontent.com/19171465/62721463-40a3da80-b9da-11e9-9faf-7752749e04ab.png)

Or on a narrower screen: 
![image](https://user-images.githubusercontent.com/19171465/62721500-531e1400-b9da-11e9-9559-74bd1d275cd3.png)

Note also the footer, which was updated in a previous commit to include ProtoSchool: 
![image](https://user-images.githubusercontent.com/19171465/62721616-9a0c0980-b9da-11e9-8423-722284676549.png)

terichadbourne commented:
Can someone who knows Hugo(?) tell me where I would put CSS that needs to apply to `content/index.html`? There's a `static/css/main.css` but it's minified. The Yeti CSS doesn't appear to have the spacing features I need to ease the from wide to narrow screens so I think I'll need to do it myself to avoid this state: 

![image](https://user-images.githubusercontent.com/19171465/62722093-a6dd2d00-b9db-11e9-8971-c893e6ad2ffc.png)

cwaring commented:
The best place for any custom css will be in [`/less/pages/home.less`](https://github.com/ipfs/website/blob/master/less/pages/home.less). I would like to check this update alongside the new homepage quiz because now we are trying to guide people down a funnel it may be better to embed these actions within the appropriate section instead.

terichadbourne commented:
Thanks @cwaring. What's your ETA on actually pushing the quiz live on the website? Unless you're within a week-ish of shipping it, I'd argue we should knock this one out in the spirit of hotfixes and let ProtoSchool enjoy some extra traffic in the meantime.

terichadbourne commented:
this was part of our efforts to address https://github.com/protoschool/protoschool.github.io/issues/260
